### ISSUE TITLE
ModuleNotFoundError: No module named 'bs4'

### ISSUE THREAD

**Issue by dapperfu** [strictdoc-project/strictdoc#336]
Installing `strictdoc` in a new virtual environment (Ubuntu 20.04) throws a `ModuleNotFoundError: No module named 'bs4'` error.

Steps to reproduce:

```
/tmp$ python3 -mvenv venv
/tmp$ source venv/bin/activate
(venv) /tmp$ pip install strictdoc
Collecting strictdoc
  Using cached strictdoc-0.0.8-py3-none-any.whl (93 kB)
Collecting XlsxWriter<2.0.0,>=1.3.7
  Using cached XlsxWriter-1.3.7-py2.py3-none-any.whl (144 kB)
Collecting jinja2==2.11.2
  Using cached Jinja2-2.11.2-py2.py3-none-any.whl (125 kB)
Collecting docutils==0.16
  Using cached docutils-0.16-py2.py3-none-any.whl (548 kB)
Collecting python-datauri<0.3.0,>=0.2.9
  Using cached python_datauri-0.2.9-py2.py3-none-any.whl (5.8 kB)
Collecting textx==2.3.0
  Using cached textX-2.3.0-py2.py3-none-any.whl (72 kB)
Collecting MarkupSafe>=0.23
  Using cached MarkupSafe-1.1.1-cp38-cp38-manylinux2010_x86_64.whl (32 kB)
Collecting Arpeggio>=1.9.0
  Using cached Arpeggio-1.10.1-py2.py3-none-any.whl (54 kB)
Installing collected packages: XlsxWriter, MarkupSafe, jinja2, docutils, python-datauri, Arpeggio, textx, strictdoc
Successfully installed Arpeggio-1.10.1 MarkupSafe-1.1.1 XlsxWriter-1.3.7 docutils-0.16 jinja2-2.11.2 python-datauri-0.2.9 strictdoc-0.0.8 textx-2.3.0
(venv) /tmp$ strictdoc 
Traceback (most recent call last):
  File "/tmp/venv/bin/strictdoc", line 5, in <module>
    from strictdoc.cli.main import main
  File "/tmp/venv/lib/python3.8/site-packages/strictdoc/cli/main.py", line 10, in <module>
    from strictdoc.core.actions.export_action import ExportAction
  File "/tmp/venv/lib/python3.8/site-packages/strictdoc/core/actions/export_action.py", line 8, in <module>
    from strictdoc.export.html.html_generator import HTMLGenerator
  File "/tmp/venv/lib/python3.8/site-packages/strictdoc/export/html/html_generator.py", line 22, in <module>
    from strictdoc.export.html.tools.html_embedded import HTMLEmbedder
  File "/tmp/venv/lib/python3.8/site-packages/strictdoc/export/html/tools/html_embedded.py", line 3, in <module>
    import bs4
ModuleNotFoundError: No module named 'bs4'
```

**Comment by stanislaw** [comment]
@dapperfu thanks for reporting this and sorry for the inconvenience.

I have just released the new 0.0.9, please give it a try.

P.S. The `bs4` was a development dependency in Poetry file. I am wondering how it could escape the CI 🤔

**Comment by dapperfu** [comment]
It works for me now. Thanks.
